specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2017-03-01-preview/sqlvm.json

```diff
@@ -1639,6 +1636,11 @@
           "description": "Virtual machine disk count.",
           "type": "integer"
         },
+        "startingDeviceId": {
```

Comment:
Could we add this to the end of the object? Otherwise, it may introduce some breaking changes. I know it's a preview.

Reply:
Done. Also just to clarify @sergey-shandar, we haven't published any CLI or Powershell with these swagger, so all changes shouldn't cause any breaking changes. After this swagger is published, I'll publish the CLI.